CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## UNRELEASED
 
-_Nothing yet..._
+### Changed
+- Bump `eslint-config-skyscanner` to `5.0.0`. Prettier is now included in `eslint-config-skyscanner`, and this package is just a wrapper around it. It's recommended to switch to `eslint-config-skyscanner` as soon as possible, as `eslint-config-skyscanner-with-prettier` will be deprecated soon.
```

Comment:
Rather than say deprecated soon, should we just say it's deprecated now and soon to be archived?

Reply:
Makes total sense, changed here and in readme.md